internal/contract/types.go

```diff
@@ -151,7 +152,7 @@ func (i *Duration) Get(obj *unstructured.Unstructured) (*metav1.Duration, error)
 	}
```

Comment:
Can you please add an assertion after `.Get` in the `Manages spec.machineTemplate.nodeDrainTimeout` unit test to ensure the new behavior?